### PR TITLE
Add HEALTHCHECK

### PR DESCRIPTION
**Pull request by giorgiosironi**
https://github.com/elifesciences/elife-base-images/blob/master/utils/assert_fpm is too complicated at the moment, so just wait on the port.

Imported a useful script, but need a way to make it easy to update from an upstream reference in `libero/`.

### CODE REVIEW

File: .travis/docker-wait-healthy [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```
Comment (giorgiosironi) [inline .travis/docker-wait-healthy:1]: should create a repository such as `libero/scripts` or similar to pull these from, with a series of `wget`. This one is going to appear in 3 repositories very soon

File: .env.dist [hunk 2]
```diff
@@ -4,3 +4,4 @@ APP_SECRET=550847c0e8dde7b2029df32625881bb1
 #TRUSTED_PROXIES=127.0.0.1,127.0.0.2
 #TRUSTED_HOSTS=localhost,example.com
 ###< symfony/framework-bundle ###
+COMPOSE_PROJECT_NAME=dummyapi
```
Comment (giorgiosironi) [inline .env.dist]: would rather not upgrade `docker-compose` on every single build

Comment (giorgiosironi) [reply .env.dist]: (to allow `dummy-api`)

Comment (thewilkybarkid) [reply .env.dist]: What's the versioning problem?

Comment (giorgiosironi) [reply .env.dist]: Older versions of `docker-compose` strip `-` from container names, while newer versions allow for `dummy-api_fpm_1` container names. Travis CI has one of the older ones.

Comment (thewilkybarkid) [reply .env.dist]: Aha, recent(ish) change (https://github.com/docker/compose/pull/5788).

Was going to suggest underscore, but the same looks to happen.

File: .travis/docker-wait-healthy (hunk 1 above)
Comment (thewilkybarkid) [reply .travis/docker-wait-healthy:1]: Though I've always avoided them, maybe a real use case for submodules? Especially given https://dependabot.com/submodules/.

File: .env.dist (hunk 2 above)
Comment (thewilkybarkid) [reply .env.dist]: Maybe could be in `.travis.yml` instead?

Comment (giorgiosironi) [reply .env.dist]: Yes, it should be equivalent. In general would default to the folder name without it.

File: .travis/docker-wait-healthy (hunk 1 above)
Comment (giorgiosironi) [reply .travis/docker-wait-healthy:1]: Would be a first-class solution, it makes more complex to deploy the source but we are not doing that favoring an artifact like a Docker image instead.

Comment (giorgiosironi) [reply .travis/docker-wait-healthy:1]: Will investigate separately not to hold up this.

File: .travis/smoke-test.sh [hunk 3]
```diff
@@ -10,5 +10,8 @@ trap finish EXIT
 
 docker-compose --file docker-compose.yaml up -d web
 docker-compose --file docker-compose.yaml exec app bin/console --version
+
+.travis/docker-wait-healthy dummyapi_app_1
```
Comment (thewilkybarkid) [inline .travis/smoke-test.sh]: Could use `COMPOSE_PROJECT_NAME`?

File: .travis.yml [hunk 4]
```diff
@@ -7,9 +7,12 @@ services: docker
 env:
     global:
         - APP_SECRET=secret
+        # update to dummy-api if docker-compose available on Travis CI accepts it:
+        - COMPOSE_PROJECT_NAME=dummyapi
         - IMAGE_TAG=$TRAVIS_COMMIT
 
 install:
+    - ln -s .env.dist .env
```
Comment (thewilkybarkid) [inline .travis.yml]: Not needed anymore?

**Review by thewilkybarkid — APPROVED** [review]

**Pull request merged**